docs/auth/faq.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: FAQ
+description: "Frequently Asked Questions about authentication modes and permissions in Hasura."
+keywords:
+  - JWT mode
+  - Webhook mode
+  - Hasura permissions
+  - NoAuth mode
+---
+
+# Frequently Asked Questions about Auth
+
+## Should I choose JWT or Webhook mode?
+
+JWT mode is the default mode and is recommended for most use cases. It's easy to set up, integrates with many 3rd party
+providers and provides a robust security model.
+
+Webhook mode is more flexible and can be useful for custom authentication scenarios. Webhook mode will be slightly
+slower than JWT mode due to the additional network request.
+
+## How do I test permissions with JWT mode?
+
+You can test permissions directly in the Hasura Console's API interface:
+
+1. Define the desired permissions for a particular Type, Model, or Command in your metadata.
+2. Create a new build of your supergraph.
+3. Make a request through the Hasura DDN Console GraphiQL API interface with an auth token that includes the required
+   session variables.
+4. Check the returned data to ensure it adheres to your permission configurations.
+
+Read more about setting up a test token in the [JWT mode tutorial](/auth/jwt/tutorials/setup-test-jwt.mdx).
+
+## How do I create a new role?
+
+A role comes into existence when it is defined in one of TypePermissions, ModelPermissions, or CommandPermissions.
+
+## How do I delete a role?
+
+To delete a role, you need to remove the role from all TypePermissions, ModelPermissions, and CommandPermissions
+objects.
+
+## How do I enable a fully open production API where any user can query anything without any auth in their queries?
```

Comment:
Should we also make a note here about marking the API as public via the console or using the CLI's `set-api-access-mode`?